src/Microsoft.AspNet.Mvc.Core/DefaultAssemblyProvider.cs

```diff
@@ -23,18 +23,21 @@ public DefaultAssemblyProvider(ILibraryManager libraryManager)
         /// Gets the set of assembly names that are used as root for discovery of
         /// MVC controllers, view components and views.
         /// </summary>
-        // DefaultControllerTypeProvider uses CandidateAssemblies to determine if the base type of a POCO controller
-        // lives in an assembly that references MVC. CandidateAssemblies excludes all assemblies from the
-        // ReferenceAssemblies set. Consequently adding WebApiCompatShim to this set would cause the ApiController to
-        /// fail this test.
+        /// <see cref="DefaultControllerTypeProvider"/> uses <see cref="CandidateAssemblies"/> to determine
+        /// if the base type of a POCO controller lives in an assembly that references MVC.
+        /// <see cref="CandidateAssemblies"/> excludes all assemblies from the <see cref="ReferenceAssemblies"/> set.
+        /// Consequently adding WebApiCompatShim to this set would cause the ApiController to fail this test.
```

Comment:
Revert this.

This isn't user-visible documentation, it's a comment for us. Adding the docs constructs like `<see .../>` has no effect.